### PR TITLE
Upgrade Checker

### PR DESCRIPTION
**Pull request by uyscuti-wiki**
This pull request contains changes and comments noting *incompatible* or *deprecated* code by the [Upgrade Checker](https://laravelshift.com/laravel-upgrade-checker). You should review each comment and commit changes to the `shift-114118` branch before merging.

Many of the potential changes found by this Shift may be automated by running the [Laravel Fixer](https://laravelshift.com/laravel-code-fixer) or tasks within the [Shift Workbench](https://laravelshift.com/workbench).

### CODE REVIEW

**Comment by uyscuti-wiki** [comment]
:warning: Shift detected the `database/factories` path being autoloaded via `classmap`. Laravel 8 namespaced _factories_ and _seeders_. If you have namespaced your factories, you may safely remove this path from your `classmap`.

**Comment by uyscuti-wiki** [comment]
:x: Shift detected that you are using Laravel's auth controllers. However, the following controllers appear to be missing or outdated. You should review these controllers by comparing yours with the [Laravel defaults](https://github.com/laravel-shift/laravel-10.x/tree/master/app/Http/Controllers/Auth) and merge any updates.

- [ ] app/Http/Controllers/Auth/ConfirmPasswordController.php

**Comment by uyscuti-wiki** [comment]
:warning: The `array_*` and `str_*` global helpers were deprecated in Laravel 5.8 and removed in Laravel 6. You should use the `Illuminate\Support\Arr` and `Illuminate\Support\Str` methods directly.

Shift found uses of these global helpers in the following files. You may use the [Laravel Fixer](https://laravelshift.com/laravel-code-fixer) to automate this change.

- [ ] database/factories/UserFactory.php

**Comment by uyscuti-wiki** [comment]
:warning: The following files differed from the default Laravel version. You should compare your file with the [latest version](https://github.com/laravel-shift/laravel-10.x/) and merge any updates.

- [ ] app/Console/Kernel.php
- [ ] app/Exceptions/Handler.php
- [ ] app/Http/Kernel.php
- [ ] artisan
- [ ] bootstrap/app.php
- [ ] phpunit.xml
- [ ] public/.htaccess

**Comment by uyscuti-wiki** [comment]
:x: The following configuration files differ from the Laravel defaults. If you are modifying configuration values directly, consider using an environment variable instead to [keep these files maintainable](https://jasonmccreary.me/articles/maintaining-laravel-config-files/). You should compare your configuration files against the [latest version](https://github.com/laravel-shift/laravel-10.x/tree/master/config) and merge any updates.

- [ ] config/app.php
- [ ] config/cache.php
- [ ] config/database.php
- [ ] config/filesystems.php
- [ ] config/mail.php
- [ ] config/services.php
- [ ] config/session.php
- [ ] config/view.php

**Comment by uyscuti-wiki** [comment]
Hi @azizullahsaeidi, could you review this and make sure everything looks good before we can merge this?

File: composer.json [hunk 1]
```diff
@@ -44,10 +44,6 @@
             "app/Support/Helpers.php",
             "app/Support/DrupalPasswordHasher.php"
         ],
-        "classmap": [
```
Comment (azizullahsaeidi) [inline composer.json:47]: `classmap` shouldn't be removed, we need `database/seeds` and `database/factories`

**Review by azizullahsaeidi — APPROVED** [review]
We need to add  `protected $model = User::class;` on UserFactory.

**Pull request closed without merging**